### PR TITLE
Jenkins 60935 Code churn metric

### PR DESCRIPTION
**Pull request by rathaya**

### CODE REVIEW

**Comment by rathaya** [comment]
Some names will be changed (LocChanges for example), as this is still a wip.

File: src/main/java/io/jenkins/plugins/forensics/miner/ForensicsBuildAction.java [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 import org.kohsuke.stapler.StaplerProxy;
+import hudson.model.Action;
 import hudson.model.Run;
 
 import io.jenkins.plugins.util.BuildAction;
 
 /**
  * Controls the live cycle of the forensics results in a job. This action persists the results of a build and displays a
- * summary on the build page. The actual visualization of the results is defined in the matching {@code summary.jelly}
+ * summary on the build page. The actual visualization of the results is defined in the matching {@code floatingBox.jelly}
```
Comment (rathaya) [inline src/main/java/io/jenkins/plugins/forensics/miner/ForensicsBuildAction.java:16]: Will revert the name change, was not intentional.

**Review by uhafner — CHANGES_REQUESTED** [review]
Can you please also check the new warnings? Otherwise Jenkins will not build the incremental release, see https://github.com/jenkinsci/forensics-api-plugin/pull/216/checks?check_run_id=1114576643 and Jenkins build.

**Review by uhafner — CHANGES_REQUESTED** [review]

File: src/main/java/io/jenkins/plugins/forensics/util/LocChanges.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+     */
+    public void updateLocChanges(final int addedLines, final int removedLines) {
+        totalLoc += addedLines;
+        totalLoc -= removedLines;
+        this.totalAddedLines += addedLines;
+        this.deletedLines += removedLines;
+    }
+
+    public int getTotalLoc() {
+        return totalLoc;
```
Comment (uhafner) [inline src/main/java/io/jenkins/plugins/forensics/util/LocChanges.java:45]: wouldn't this be the same (then the field is not used)?
```suggestion
        return addedLines - removedLines;
```

**Pull request merged**